event_routing_backends/processors/xapi/constants.py

```diff
@@ -46,6 +46,7 @@
 XAPI_CONTEXT_STARTING_POSITION = 'http://id.tincanapi.com/extension/starting-position'
 XAPI_CONTEXT_ENDING_POSITION = 'http://id.tincanapi.com/extension/ending-point'
 XAPI_CONTEXT_COMPLETION_THRESHOLD = 'https://w3id.org/xapi/video/extensions/completion-threshold'
+XAPI_CONTEXT_SESSION_ID = 'https://w3id.org/xapi/video/extensions/session-id'
```

Comment:
It feels slightly weird that we're using the video extension for all of the interactions, but I'm not seeing one that seems like a better fit. The CMI5 one is the only other candidate and is documented as specific to AUs, so I think this is the right choice, I'm just noting here that I looked into it.